### PR TITLE
ncm-sudo: Fix generation of parameter lists

### LINKED ISSUE #460: ncm-sudo: generating incorrect Defaults lines with multiple options
Given

```pan
"/software/components/sudo/general_options" = list(
    nlist("options", nlist(
                     "requiretty",true,
                     "env_reset", true,
)));
```

ncm-sudo generates (as seen by `ncm-ncd --configure --verbose sudo`)
```conf
Defaults	requiretty	    env_reset
```
which is incorrect, it should be either
```conf
Defaults   requiretty
Defaults   env_reset
```
or 
```conf
Defaults  requiretty, env_reset
```

### PR DESCRIPTION
**Pull request by jrha**
Specifying multiple options was possible, but untested by the unit tests, the delimiter was incorrectly specified as a tab character rather than a comma.

Fixes #460 as reported by @drossy.

### CODE REVIEW

File: ncm-sudo/src/main/perl/sudo.pm [hunk 1]
```diff
@@ -138,48 +143,47 @@ sub generate_aliases {
...
+                if ($opts{$b}->getValue eq 'true') {
+                    $ln .= "$b,";
+                } else {
+                    $ln .= "!$b,";
+                }
+            }
+        }
+        foreach my $o (INT_OPTS, STRING_OPTS) {
+            $ln .= "$o=" . $opts{$o}->getValue . ","
+            if defined $opts{$o};
```
Comment (stdweird) [inline ncm-sudo/src/main/perl/sudo.pm]: indent

File: ncm-sudo/src/main/perl/sudo.pm [hunk 2]
```diff
@@ -138,48 +143,47 @@ sub generate_aliases {
...
+        } elsif (defined $def{PRIVILEGE_HOST ()}) {
+            $ln = "@" . $def{PRIVILEGE_HOST ()}->getValue . "\t";
+        } elsif (defined $def{PRIVILEGE_CMD ()}) {
+            $ln = "!" . $def{PRIVILEGE_CMD ()}->getValue . "\t";
+        } else {
+            $ln = "\t";
+        }
+        my %opts = $def{PRIVILEGE_OPTS()}->getHash;
+        foreach my $b (BOOLEAN_OPTS) {
+            if (defined $opts{$b}) {
```
Comment (stdweird) [inline ncm-sudo/src/main/perl/sudo.pm]:
```perl
$ln .= ($opts{$b}->getValue eq 'true' ? '' : '!' ). "$b,"
    if (defined $opts{$b});
```

**Comment by stdweird** [comment]
@jrha minor remarks, looks OK

**Comment by jrha** [comment]
Thanks!

**Comment by stdweird** [comment]
@jrha LGTM

**Pull request merged**